src/core_plugins/kibana/public/dashboard/panel/panel_header.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { KuiKeyboardAccessible } from 'ui_framework/components';
+import { PanelOptionsMenu } from './panel_options_menu';
+
+export class PanelHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getOptionsDropDown() {
+    return (
+      <PanelOptionsMenu
+        onEditPanel={this.props.onEditPanel}
+        onDeletePanel={this.props.onDeletePanel}
+        onToggleExpandPanel={this.props.onToggleExpand}
+        isExpanded={this.props.isExpanded}
+      />
+    );
+  }
+
+  getExpandToggle() {
+    const { isExpanded } = this.props;
+    const classes = classNames('kuiIcon', null, { 'fa-expand': !isExpanded, 'fa-compress': isExpanded });
+    return (
+      <KuiKeyboardAccessible>
+        <a
+          className="kuiMicroButton"
+          aria-label="Expand panel"
```

Comment:
Could you change this into "Toggle panel options", "toogle" is more common used, since `aria-expanded` will take care of announcing the actual state to a screen reader.

Reply:
This button doesn't actually expand the options menu, it makes the panel take up the full screen.  How about `Maximize panel`?